vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java

```diff
@@ -15,11 +15,21 @@
  */
 package com.vaadin.flow.theme.lumo;
 
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.dependency.NpmPackage;
+
 /**
  * Contains the definition for all the CSS utility classes provided by Lumo.
  */
+@NpmPackage(value = "@vaadin/vaadin-lumo-styles", version = "23.2.0-alpha2")
+@JsModule("@vaadin/vaadin-lumo-styles/utility.js")
+@JsModule("./lumo-utility.ts")
 public final class LumoUtility {
 
+    private static String notConstant(String value) {
```

Comment:
Let's add a comment based on https://github.com/vaadin/flow-components/pull/3383#issuecomment-1205044363 to explain the purpose of this.

Reply:
Done